prow/plugins/trigger/trigger.go

```diff
@@ -135,11 +135,11 @@ func handlePush(pc plugins.PluginClient, pe github.PushEvent) error {
 	return handlePE(getClient(pc), pe)
 }
 
-// trustedUser returns true if user is trusted in repo.
+// TrustedUser returns true if user is trusted in repo.
 //
 // Trusted users are either repo collaborators, org members or trusted org members.
 // Whether repo collaborators and/or a second org is trusted is configured by trigger.
-func trustedUser(ghc githubClient, trigger *plugins.Trigger, user, org, repo string) (bool, error) {
+func TrustedUser(ghc githubClient, trigger *plugins.Trigger, user, org, repo string) (bool, error) {
```

Comment:
So instead of using `githubClient`, pass around a new `trustedUserClient` (or something similarly named) that is a subset of the `githubClient` interface. (Only `IsCollaborator` and `IsMember` are needed.)